setup.cfg

```diff
@@ -11,7 +11,7 @@ combine_as_imports = True
 force_grid_wrap = 0
 include_trailing_comma = True
 known_first_party = httpx,httpxprof,tests
-known_third_party = brotli,certifi,chardet,click,cryptography,h11,h2,hstspreload,nox,pytest,requests,rfc3986,setuptools,tqdm,trio,trustme,uvicorn
+known_third_party = brotli,certifi,chardet,click,cryptography,h11,h2,hstspreload,hypercorn,nox,pytest,requests,rfc3986,setuptools,tqdm,trio,trustme,uvicorn
```

Comment:
We can remove uvicorn from this list?

Reply:
Uvicorn is still used as a test server - for the HTTP/1.1 parts. Is the expectation that Hypercorn replaces Uvicorn?